packages/@lwc/engine-dom/typings/observedAttributes.d.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+// TypeScript seems to be missing this definition from dom.d.ts. It can be removed when
+// this is fixed: https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1338
+declare interface CustomElementConstructor {
+    observedAttributes?: string[];
```

Comment:
I could not figure out how to extend the original `CustomElementConstructor` from `lib.dom` but here is an alternative.
```suggestion
     new (...params: any[]): HTMLElement;
    observedAttributes?: string[];
```

Reply:
What is the benefit of adding this line? Sorry, I'm not super adept at TypeScript. 😕

Reply:
oh, `CustomElementConstructor` is being redefined here and that constructor signature was missing. For the sake of completeness of the interface, I suggested adding it